### ISSUE TITLE
Add icons for work objects?

### ISSUE THREAD

**Issue by hofstef** [WPS/domain-story-modeler#17]
The standard icon set was compiled to express typical "office work" and should cover a lot of domains. Are there some icons missing that are universally applicable?

**Comment by sherl0cks** [comment]
@hofstef the icon set is a good start, but it would be really nice to be able to add one's own icon. Is that in the realm of possibility? I would be interested in submitting a PR to that end.

**Comment by hofstef** [comment]
Customizing icons (or icon sets) is indeed important and I collected some ideas for this in issue #7 .
This issue (#17) is more about the default icon set and if there is something missing.
Concerning a possible PR: We are open for that, but I think it is best if we do some cleanup first. There is also branch where we experimented with uploading SVG icons for individual work objects/actors. It works, but is probably not a solution for the most pressing problem - having a customizable icon set.
